HW_20/test_api.py

```diff
@@ -0,0 +1,45 @@
+from http import HTTPStatus
+from HW_20.data.user_data import User
+from api_collections.user_collection import UserAPI
+import json
+
+
+def test_get_user_200():
+    getresponse = UserAPI().get_user(7)
+    assert getresponse.status_code == HTTPStatus.OK, f'Status code is not as expected\nActual ' \
+                                                     f'{getresponse.status_code}' \
+                                                     f'\nExpected: {HTTPStatus.OK}'
+
+
+def test_post_user_201():
+    postresponse = UserAPI().post_user('morpheus', 'leader')
+    assert postresponse.status_code == HTTPStatus.CREATED, f'Status code is not as expected\nActual ' \
+                                                           f'{postresponse.status_code}' f'\nExpected: ' \
+                                                           f'{HTTPStatus.CREATED}'
+
+
+def test_put_user_200():
+    response = UserAPI().put_user('morpheus', 'zion resident')
+    assert response.status_code == HTTPStatus.OK, f'Status code is not as expected\nActual {response.status_code}' \
```

Comment:
is it possible to get this updated user after your action?
If yes it will be good to check if your update works fine...if not just remember about this for your future work